### DISCUSSION TITLE
How to exclude resources from open api generated file

### DISCUSSION THREAD

**Discussion by ennishol** [quarkusio/quarkus#37847] [Community]
Hi,
quarkus generates open api specification file for all found endpoints. I want to exclude some or only include those with  org.eclipse.microprofile.openapi.annotations.* annotations.
is it possible?
Thanks!

**Comment by ramiswailem** [comment]
@Operation(hidden = true)

**Reply by ennishol** [reply]
@ramiswailem it works but then every method has to be annotated. I'd like to exclude a complete resource or allow only some resources.

I've implemented mp.openapi.filter and it works, but it would be nice to have something simpler

**Comment by mschorsch** [comment]
@MikeEdgar Can you help here?

**Comment by MikeEdgar** [comment]
Using `@Operation(hidden = true)` will work or you could use some combination of the include/exclude configuration properties starting with `mp.openapi.scan.` [1]



[1] https://download.eclipse.org/microprofile/microprofile-open-api-3.1.1/microprofile-openapi-spec-3.1.1.html#_core_configurations

**Comment by ennishol** [comment]
@MikeEdgar  thank you!
